lib/Provider/NotificationProvider.php

```diff
@@ -97,17 +87,6 @@ public function getDescription(): string {
 	public function getTemplate(IUser $user): Template {
 		$token = $this->tokenManager->generate($user->getUID());
 
-		//Send notificcation
```

Comment:
but where is the notification sent now? :wink:

Reply:
https://github.com/nickv-nextcloud/twofactor_nextcloud_notification/pull/28/files#diff-b38e66c692640d90ac2fa5c6bd42340bR54

via https://github.com/nickv-nextcloud/twofactor_nextcloud_notification/pull/28/files#diff-fd9f67990eaf70263df0efd21a4a5a19R99

Reply:
Right, missed this path!